Exec/science/nova/inputs_nova_t7

```diff
@@ -102,14 +108,14 @@ integrator.abort_on_failure = 0
 
 amr.refinement_indicators = dens temp edot
 
-amr.refine.dens.max_level = 5
+amr.refine.dens.max_level = 3
 amr.refine.dens.value_greater = 1.0e2
```

Comment:
you don't need to change these -- you can leave them high -- it just means that you always want these indicators to get up to max_level

Reply:
I restored the values to their original value.